ulog/README.md

```diff
@@ -53,16 +51,19 @@ For example, the following piece of code:
 ```go
 package main
 
-import "go.uber.org/fx/ulog"
+import (
+  "context"
 
+  "go.uber.org/fx/ulog"
+)
 func main() {
-  log := ulog.Logger()
-  log.Info("My info message")
-  log.Info("Info with context", "customer_id", 1234)
+  log := ulog.Logger(context.Background())
```

Comment:
The use of context is necessary to demonstrate context based logging (with tracer, etc available in context). Perhaps writing it in main is not wise. Will update it to some handler function that provides a context.